opencl/source/sharings/gl/linux/gl_buffer_linux.cpp

```diff
@@ -168,14 +199,20 @@ GraphicsAllocation *GlBuffer::createGraphicsAllocation(Context *context, unsigne
             auto helper = context->getDevice(0)->getRootDeviceEnvironment().getGmmHelper();
             graphicsAllocation->setDefaultGmm(new Gmm(helper, bufferInfo.pGmmResInfo));
         }
+        else
+        {
+            auto helper = context->getDevice(0)->getRootDeviceEnvironment().getGmmHelper();
+            StorageInfo si = {};
```

Comment:
```suggestion
            StorageInfo storageInfo = {};
```

please use meaningful name